wagtail/admin/templates/wagtailadmin/shared/workflow_status.html

```diff
@@ -24,11 +24,13 @@ <h2>
         {% trans 'Approved task' as approved_title %}
         {% trans 'Incomplete task' as incomplete_title %}
         {% for task in workflow_tasks %}
-            {% if task.status == 'approved' %}
-                {% wagtail_icon name="success" title=approved_title %}
-            {% else %}
-                {% wagtail_icon name="radio-empty" title=incomplete_title %}
-            {% endif %}
+            <span data-wagtail-tooltip="{{ task.name }}: {{ task.status_display }}">
```

Comment:
I think this wouldn't be screen reader accessible - possibly we could follow one of the approaches here, under toggletips? https://inclusive-components.design/tooltips-toggletips/

Reply:
At the very least make this info accessible on the icon instead, now we have it in the context